assets/aml-benchmark/components/batch_output_formatter/spec.yaml

```diff
@@ -71,7 +71,7 @@ outputs:
   unsafe_content_blocked_requests:
     type: uri_file
 code: ../src
-environment: azureml://registries/azureml/environments/model-evaluation/labels/latest
+environment: azureml://registries/azureml/environments/aoai-evaluation/labels/latest
```

Comment:
I'd expect environment to be released first and tested before updating the components?

Reply:
removed component update